hydra-node/src/Hydra/Node.hs

```diff
@@ -203,11 +204,13 @@ processEffect HydraNode{hn, oc = Chain{postTx}, server, eq, env = Environment{pa
 -- alternative implementation
 data EventQueue m e = EventQueue
   { putEvent :: e -> m ()
-  , putEventAfter :: DiffTime -> e -> m ()
-  , nextEvent :: m e
+  , putEventAfter :: DiffTime -> Queued e -> m ()
```

Comment:
I like the `Queued` wrapper. It makes it clear that this is should actually be called `reenqueueAfter`. No need to change now.

Reply:
Yes, it's for another PR